{
  "ID": "e53ad624-0c30-40a8-b8fe-61d600d2f2fb",
  "Root": {
    "alive": true,
    "content": [
      "37b39e4a-aedb-4d50-8ef6-f41614797e13"
    ],
    "created_by": "bb760e2d-d679-4b64-b2a9-03005b21870a",
    "created_time": 1550457392211,
    "id": "e53ad624-0c30-40a8-b8fe-61d600d2f2fb",
    "ignore_block_count": true,
    "last_edited_by": "bb760e2d-d679-4b64-b2a9-03005b21870a",
    "last_edited_time": 1550481660000,
    "parent_id": "0a7e3f26-8b3b-412d-9432-984541779d68",
    "parent_table": "block",
    "properties": {
      "title": [
        [
          "Custom exception"
        ]
      ]
    },
    "type": "page",
    "version": 6,
    "content_resolved": [
      {
        "alive": true,
        "created_by": "bb760e2d-d679-4b64-b2a9-03005b21870a",
        "created_time": 1550457392208,
        "id": "37b39e4a-aedb-4d50-8ef6-f41614797e13",
        "ignore_block_count": true,
        "last_edited_by": "bb760e2d-d679-4b64-b2a9-03005b21870a",
        "last_edited_time": 1550481660000,
        "parent_id": "e53ad624-0c30-40a8-b8fe-61d600d2f2fb",
        "parent_table": "block",
        "properties": {
          "language": [
            [
              "Dart"
            ]
          ],
          "title": [
            [
              "class CustomException implements Exception {\n  String cause;\n  CustomException(this.cause);\n}\n\nvoid main() {\n  try {\n    throwException();\n  } on CustomException {\n    print(\"custom exception is been obtained\");\n  }\n}\n\nthrowException() {\n  throw new CustomException('This is my first custom exception');\n}"
            ]
          ]
        },
        "type": "code",
        "version": 3,
        "code": "class CustomException implements Exception {\n  String cause;\n  CustomException(this.cause);\n}\n\nvoid main() {\n  try {\n    throwException();\n  } on CustomException {\n    print(\"custom exception is been obtained\");\n  }\n}\n\nthrowException() {\n  throw new CustomException('This is my first custom exception');\n}",
        "code_language": "Dart"
      }
    ],
    "title": "Custom exception"
  },
  "Users": [
    {
      "email": "kkowalczyk@gmail.com",
      "family_name": "Kowalczyk",
      "given_name": "Krzysztof",
      "id": "bb760e2d-d679-4b64-b2a9-03005b21870a",
      "locale": "en",
      "mobile_onboarding_completed": true,
      "onboarding_completed": true,
      "profile_photo": "https://s3-us-west-2.amazonaws.com/public.notion-static.com/2dcaa66c-7674-4ff6-9924-601785b63561/head-bw-640x960.png",
      "time_zone": "America/Los_Angeles",
      "version": 18
    }
  ],
  "Tables": null
}